scripts/auth.js

```diff
@@ -49,3 +57,20 @@ async function formSubmit(event) {
     form.classList.add('was-validated');
   }
 }
+
+function toggleNavLinks(username) {
+  const signInLink  = document.getElementById(SIGNIN_LINK_ID);
+  const signOutLink = document.getElementById(SIGNOUT_LINK_ID);
+  const profileLink = document.getElementById(PROFILE_CONTAINER_ID);
+
+  if (username) {
+    signInLink.hidden  = true;
+    signOutLink.hidden = false;
+    profileLink.href   = `/users/${username}`;
```

Comment:
If we are using their username as a url, we need to think of how to sanitize it.
For example, what if they do something like [this](https://en.wikipedia.org/wiki/Directory_traversal_attack)?

Reply:
b/161373767
good looks! maybe we can limit usernames to alphanumeric and underscores